include/aspect/utilities.h

```diff
@@ -143,6 +143,12 @@ namespace aspect
          */
         bool check_values_per_key;
 
+        /**
+        * Whether to define the phase properties based
+        * on the exact values, or the value differences.
+        */
+        bool use_differences_instead_of_values;
```

Comment:
I found the function 'parse_map_to_double_array" has been rewritten in previous PRs. So I have to the add this attribute to the class they implemented.

Reply:
I would much prefer to have this information outside of the `parse` function. The reason is the following:
This function can be used to parse arbitrary maps for any input parameter in ASPECT. We tried very hard to make it this generic. However, this new property is extremely specific and makes not much sense for other properties except the ones in material models. I understand that you need a way to modify the parsing if differences rather than values are expected, but I think this can be achieved in a more natural way (see comments below).